web/profiles/custom/yalesites_profile/modules/custom/ys_core/templates/ys-breadcrumb-block.html.twig

```diff
@@ -3,12 +3,13 @@
  # - items (array)
  #   - title: Link title
  #   - url: Link url
+ #   - is_active: (bool) is current page
  #}
 
 <ul>
   {% for item in items %}
     <li>
-      {% if item.url %}
+      {% if not item.is_active %}
```

Comment:
There are indentation issues here. And I'm wondering if we should make this a little more obvious by removing the `not`. So:

```
{% if item.is_active %}
  {{ item.title }}
{% else %}
  <a href="{{ item.url }}">{{ item.title }}</a>
{% endif %}
```